source/ed/build_helper.cpp

```diff
@@ -92,49 +74,43 @@ VJ::VJ(builder & b, const std::string &line_name, const std::string &validity_pa
     } else {
         route = it->second->route_list.front();
     }
-    if (jp_uri.empty()) {
-        jp_uri = route->uri + ":0";
-    }
-    auto jp = get_or_create_journey_pattern(b, jp_uri);
 
-    route->journey_pattern_list.push_back(jp);
-    jp->route = route;
+    if (is_frequency) {
+        auto f_vj = std::make_unique<nt::FrequencyVehicleJourney>();
+        f_vj->idx = route->frequency_vehicle_journey_list.size() - 1;
```

Comment:
That's in the old version line 127. It seems buggy as we should use data.vehicle_journeys.size(). But we may reindex that in a data finalizer, thus it's not really important.